.github/workflows/check-wasm.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         with:
           target: wasm32-unknown-unknown
           components: rust-src
-          toolchain: nightly-2024-01-01
+          toolchain: nightly-2024-07-25
```

Comment:
cargo-stylus refused to compile on `nightly-2024-01-01` this time. There won't be a problem, when we will merge these changes after [pr with nightly fix](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/375)